example/data/rules/L2_HR_PIXC_kinesisRule.json

```diff
@@ -7,8 +7,7 @@
   "provider": "PODAAC_SWOT",
   "rule": {
     "type": "kinesis",
-    "value": "arn:aws:kinesis:{{awsRegion}}:{{awsAccountId}}:stream/{{streamName}}",
-    "arn": "arn:aws:kinesis:{{awsRegion}}:{{awsAccountId}}:stream/{{streamName}}"
```

Comment:
Removes un-needed and incorrect "arn" key from the rule template.  Was a result of this conversation:  https://github.com/nasa/cumulus/pull/496#discussion_r209306592